packages/firestore/src/local/local_store.ts

```diff
@@ -918,7 +936,39 @@ export class LocalStore {
     return promiseChain;
   }
 
-  getActiveClients(): Promise<ClientId[]> {
-    return this.persistence.getActiveClients();
+  /**
+   *  Stores the mutation keys associated with the given batch ID. The mutation
+   *  keys can later be retrieved via `lookupMutationDocuments`.
+   */
+  // PORTING NOTE: Multi-tab only.
+  private insertMutationKeys(batchId: BatchId, keys: DocumentKeySet): void {
+    this.localMutationBatchKeys[batchId] = keys;
+  }
+
+  /**
+   * Looks up mutation keys for the given batch ID. If the mutation is not
+   * known locally (e.g. when another client added it), this method looks up the
+   * mutation from persistence.
+   */
+  // PORTING NOTE: Multi-tab only.
+  private lookupAndStoreMutationKeys(
```

Comment:
lookupAndStore seems odd to me. :)  Perhaps lookupAndCache...?

Reply:
No longer applicable, but I did rename it from `lookupAndCache` to `lookupAndStore`earlier since `cache` to me hinted to strongly that I might have implemented some cleanup magic.